docs/core/diagnostics/event-counter-perf.md

```diff
@@ -0,0 +1,100 @@
+
+```csharp
+// Give your event sources a descriptive name using the EventSourceAttribute, otherwise the name of the class is used.
+[EventSource(Name = "Samples-EventCounterDemos-Minimal")]
+public sealed class MinimalEventCounterSource : EventSource
+{
+    // Define the singleton instance of the event source
+    public static MinimalEventCounterSource Log = new MinimalEventCounterSource();
+    private EventCounter RequestCounter;
+
+    private MinimalEventCounterSource() : base(EventSourceSettings.EtwSelfDescribingEventFormat)
+    {
+        RequestCounter = new EventCounter("request", this);
+    }
+
+    // Indicate that a request for a URL was made which took a particular amount of time
+    public void Request(string url, float elapsedMSec)
+    {
+        // Notes:
+        //   1. the event ID passed to WriteEvent (1) corresponds to the (implied) event ID
+        //      assigned to this method. The event ID could have been explicitly declared
+        //      using an EventAttribute for this method
+        //   2. Each counter supports a single float value, so conceptually it maps to a single
+        //      measurement in the code.
+        //   3. You don't have to have log with WriteEvent if you don't think you will ever care about details
+        //       of individual requests (that counter data is sufficient).
+
+        // This logs it to the event stream if events are on.
+        WriteEvent(1, url, elapsedMSec);
+        // This adds it to the PerfCounter called 'Request' if PerfCounters are on
+        RequestCounter.WriteMetric(elapsedMSec);
+    }
+}
+```
+
+The <xref:System.Diagnostics.Tracing.EventSource.WriteEvent%2A?displayProperty=nameWithType> line is the <xref:System.Diagnostics.Tracing.EventSource> part and is not part of <xref:System.Diagnostics.Tracing.EventCounter>, it was written to show that you can log a message together with the event counter.
+
+You logged the metric to the <xref:System.Diagnostics.Tracing.EventCounter>, but unless you can actually get the statistics out of it, it is not useful. To get the statistics, you need to enable the <xref:System.Diagnostics.Tracing.EventCounter> by setting off a timer how frequently you want the events, as well as a listener to capture the events, to do that, you can use PerfView.
+
+There is an extra keyword that you will need to specify the turn on the EventCounters.
+
+```
+PerfView /onlyProviders=*Samples-EventCounterDemos-Minimal:EventCounterIntervalSec=1 collect
+```
+
+> [!IMPORTANT]
+> The `EventCounterIntervalSec` segment is used to indicate the frequency of the sampling.
+
+As usual, turn on PerfView, and then run the sample code - we get have something like this.
```

Comment:
```suggestion
Turn on PerfView, and then run the sample code - we get have something like this.
```

What sample code do we run or how do we run it? the preceding sample doesn't have a Main method.